pkgs/applications/office/scribus/default.nix

```diff
@@ -61,12 +68,21 @@ mkDerivation rec {
     # For Poppler 22.04
     (fetchpatch {
       url = "https://github.com/scribusproject/scribus/commit/f2237b8f0b5cf7690e864a22ef7a63a6d769fa36.patch";
-      sha256 = "FXpLoX/a2Jy3GcfzrUUyVUfEAp5wAy2UfzfVA5lhwJw=";
+      sha256 = "xOOi200iD7NL7dHB00/GnFJ0TUhlH9XouO7QGMItOlY=";
+      postFetch = "${dos2unix}/bin/dos2unix $out";
     })
+    # For Poppler 22.05
+    # Base off of https://github.com/scribusproject/scribus/commit/8acd29e97813b9132e3b51b2f05e8fac65819ed7.patch
+    ./poppler-22.09.0.patch
   ];
 
+  postPatch = ''
+    unix2dos scribus/plugins/import/pdf/slaoutput.cpp
```

Comment:
Why is not it sufficient to make the third patch apply without converting everything else?